<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ipv6Widget</class>
 <widget class="QWidget" name="Ipv6Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="0" column="3">
    <widget class="QLabel" name="cidrLabel">
     <property name="text">
      <string>/</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="ipv6Address">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>2001:0db8:85a3::8a2e::0370:7334</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="subnetCountLabel">
     <property name="text">
      <string>Subnet Count</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="4">
    <widget class="QSpinBox" name="cidr">
     <property name="maximum">
      <number>128</number>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="subnetCount">
     <item>
      <property name="text">
       <string>1 (2^0)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="ipv6AddressLabel">
     <property name="text">
      <string>IPv6 Address</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="5">
    <widget class="QTreeWidget" name="subnetsTable">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectItems</enum>
     </property>
     <property name="rootIsDecorated">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="headerHidden">
      <bool>false</bool>
     </property>
     <attribute name="headerMinimumSectionSize">
      <number>0</number>
     </attribute>
     <attribute name="headerDefaultSectionSize">
      <number>140</number>
     </attribute>
     <column>
      <property name="text">
       <string>No.</string>
      </property>
      <property name="textAlignment">
       <set>AlignTrailing|AlignVCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Routing Prefix</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Range</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Prefix Length (CIDR)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Host Count</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="calculate">
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ipv6Address</tabstop>
  <tabstop>cidr</tabstop>
  <tabstop>subnetCount</tabstop>
  <tabstop>calculate</tabstop>
  <tabstop>subnetsTable</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
